core/base/v7/inc/ROOT/RError.hxx

```diff
@@ -215,7 +217,7 @@ public:
          fError->AppendToMessage(" (unchecked RResult access!)");
          throw RException(*fError);
       }
-      return fValue;
+      return std::move(fValue);
```

Comment:
Tricky problem.  The issue with this is that you can call `Get()` at most once.  That might be ok but in this case I think

- we should test if fValue has been moved before (and fail if that was the case)
- Rename the method, perhaps to `Move`?

Reply:
I like these names, let's rename our methods to `Unwrap` and `Inspect`

Reply:
I opened a PR for these changes only here: #5985